ckan/lib/helpers.py

```diff
@@ -1372,11 +1372,22 @@ def linked_user(user, maxlength=0, avatar=20):
 
 
 @core_helper
+@maintain.deprecated("helpers.group_name_to_title()"
+                     " is deprecated and will be removed"
+                     " in a future version of CKAN.")
 def group_name_to_title(name):
+    """
+    Deprecated: will be removed in a future version of CKAN.
+    """
     group = model.Group.by_name(name)
-    if group is not None:
-        return group.display_name
-    return name
+    if group is None:
+        return name
+    group_dict = logic.get_action(u'%s_show' % group.type)(
+                    {}, {u'id': group.id})
```

Comment:
we should pass all the flags that will make this call faster, i.e. `include_dataset_count=False, include_users=False, include_groups=False, include_tags=False, include_followers=False`